test/lexers/javascript-lexer.test.js

```diff
@@ -109,15 +100,12 @@ describe('JavascriptLexer', () => {
     ])
     done()
   })
-
-  describe('supports the acorn-stage3 plugin', () => {
-
-    it('supports dynamic imports', (done) => {
-      const Lexer = new JavascriptLexer({ acorn: { ecmaVersion: 6, plugins: { stage3: true } } })
-      const content = 'import("path/to/some/file").then(doSomethingWithData)'
-      Lexer.extract(content)
+  describe('supports additional plugins via injector option', () => {
+    it('provided injectors are called with acorn', (done) => {
```

Comment:
the name of the test should read as valid english. For instance: it... `injects provided acorn plugins`